src/script/components/Emoji/EmojiItem.tsx

```diff
@@ -33,15 +34,15 @@ interface EmojiItemProps {
 const EmojiItem: FC<EmojiItemProps> = ({emoji, onClick, onMouseEnter, selectedEmoji = false}) => (
   <button
     type="button"
```

Comment:
@arjita-mitra I think this `type=button` is not necessary here. Is it?

Reply:
I believe it's only needed for `input`, am I correct?

Reply:
Oh actually I found my answer and it is needed :)

Reply:
Same for `tabIndex={0}`, looking like leftovers from changing the component from `div` to `button`

Reply:
Oh correct the tabindex is not needed, good catch 👍